ts/Core/Series/DataLabel.ts

```diff
@@ -31,6 +31,8 @@ import type SVGLabel from '../Renderer/SVG/SVGLabel';
 import type SVGPath from '../../Core/Renderer/SVG/SVGPath';
 import type AnimationOptions from '../Animation/AnimationOptions';
 
+import TextPath from '../../Extensions/TextPath.js';
```

Comment:
By importing TextPath into DataLabel.ts, we still make it part of `highcharts.js`, which was what we wanted to avoid.

Instead, we need to use an event in DataLabel.ts that the TextPath module can subscribe to, so that the module is not a dependency.